[2017-08-23 09:21:01] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php","Line":176,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(RouterListener.php:176)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(RouterListener.php:176)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(EventDispatcher.php:184)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(EventDispatcher.php:46)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(UrlMatcher.php:108)\n at Symfony.Component.Routing.Router.matchRequest(Router.php:256)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(RouterListener.php:154)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(EventDispatcher.php:184)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(EventDispatcher.php:46)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-08-23 09:21:01] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/EventListener/RouterListener.php","Line":176,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(RouterListener.php:176)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(RouterListener.php:176)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(EventDispatcher.php:184)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(EventDispatcher.php:46)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(UrlMatcher.php:108)\n at Symfony.Component.Routing.Router.matchRequest(Router.php:256)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(RouterListener.php:154)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(EventDispatcher.php:184)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(EventDispatcher.php:46)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-08-23 09:21:01] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/bootstrap/glyphicons-halflings-regular.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-08-23 10:33:14] rest_api.operation.DEBUG: Response to controller Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::getImportFilesAction has been returned. {"status_code":403,"headers":{"cache-control":["no-cache"],"content-type":["application/json; charset=utf-8"]}} {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":110,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelResponse"}
[2017-08-23 10:33:14] rest_api.operation.ERROR: An error has been returned by Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::getImportFilesAction {"status_code":403,"content":"{\"error\":\"Username incorrect\"}"} {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":119,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelResponse"}
[2017-08-23 10:51:24] rest_api.operation.DEBUG: Request to controller Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::getImportFilesAction has been received. {"request":{"url":"/app_dev.php/rest/operation/v1/importfiles","format":"json","client_ip":"192.168.128.1","content_type":"form","method":"GET","query_string":null,"attributes":{"media_type":"application/json","_controller":"Pivot88\\Bundle\\RestApiBundle\\Controller\\ImportfilesController::getImportFilesAction","_format":"json","version":"v1","_route":"api_1_get_import_files","_route_params":{"_format":"json","version":"v1"}},"query":[],"headers":{"accept":"*/*","content-type":"application/x-www-form-urlencoded","referer":null},"request":[]}} {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":90,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelController"}
[2017-08-23 10:57:09] rest_api.operation.DEBUG: Request to controller Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::postImportfilesAction has been received.  {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":90,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelController"}
[2017-08-23 10:57:09] rest_api.operation.DEBUG: Response to controller Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::postImportfilesAction has been returned. {"status_code":403,"headers":{"cache-control":["no-cache"],"x-debug-token":["13e9ff"],"x-debug-token-link":["http://localhost:8080/app_dev.php/_profiler/13e9ff"],"access-control-allow-origin":["chrome-extension://fhbjgbiflinjbdggehcddcbncdddomop"]}} {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":110,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelResponse"}
[2017-08-23 10:57:09] rest_api.operation.ERROR: An error has been returned by Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::postImportfilesAction {"status_code":403,"content":"{\"error\":{\"code\":403,\"message\":\"Forbidden\",\"exception\":[{\"message\":\"Access Denied.\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\Exception\\\\AccessDeniedHttpException\",\"trace\":[{\"namespace\":\"\",\"short_class\":\"\",\"class\":\"\",\"type\":\"\",\"function\":\"\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/Security\\/Http\\/Firewall\\/ExceptionListener.php\",\"line\":119,\"args\":[]},{\"namespace\":\"Symfony\\\\Component\\\\Security\\\\Http\\\\Firewall\",\"short_class\":\"ExceptionListener\",\"class\":\"Symfony\\\\Component\\\\Security\\\\Http\\\\Firewall\\\\ExceptionListener\",\"type\":\"->\",\"function\":\"handleAccessDeniedException\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/Security\\/Http\\/Firewall\\/ExceptionListener.php\",\"line\":97,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"],[\"object\",\"Symfony\\\\Component\\\\Security\\\\Core\\\\Exception\\\\AccessDeniedException\"]]},{\"namespace\":\"Symfony\\\\Component\\\\Security\\\\Http\\\\Firewall\",\"short_class\":\"ExceptionListener\",\"class\":\"Symfony\\\\Component\\\\Security\\\\Http\\\\Firewall\\\\ExceptionListener\",\"type\":\"->\",\"function\":\"onKernelException\",\"file\":null,\"line\":null,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"],[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Debug\\\\TraceableEventDispatcher\"]]},{\"namespace\":\"\",\"short_class\":\"\",\"class\":\"\",\"type\":\"\",\"function\":\"call_user_func\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/EventDispatcher\\/Debug\\/WrappedListener.php\",\"line\":61,\"args\":[[\"array\",[[\"object\",\"Symfony\\\\Component\\\\Security\\\\Http\\\\Firewall\\\\ExceptionListener\"],[\"string\",\"onKernelException\"]]],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"],[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Debug\\\\TraceableEventDispatcher\"]]},{\"namespace\":\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\",\"short_class\":\"WrappedListener\",\"class\":\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\",\"type\":\"->\",\"function\":\"__invoke\",\"file\":null,\"line\":null,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"],[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\ContainerAwareEventDispatcher\"]]},{\"namespace\":\"\",\"short_class\":\"\",\"class\":\"\",\"type\":\"\",\"function\":\"call_user_func\",\"file\":\"\\/srv\\/app\\/cache\\/dev\\/classes.php\",\"line\":1858,\"args\":[[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"],[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\ContainerAwareEventDispatcher\"]]},{\"namespace\":\"Symfony\\\\Component\\\\EventDispatcher\",\"short_class\":\"EventDispatcher\",\"class\":\"Symfony\\\\Component\\\\EventDispatcher\\\\EventDispatcher\",\"type\":\"->\",\"function\":\"doDispatch\",\"file\":\"\\/srv\\/app\\/cache\\/dev\\/classes.php\",\"line\":1773,\"args\":[[\"array\",[[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\"],[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\"],[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\"],[\"object\",\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\WrappedListener\"]]],[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"]]},{\"namespace\":\"Symfony\\\\Component\\\\EventDispatcher\",\"short_class\":\"EventDispatcher\",\"class\":\"Symfony\\\\Component\\\\EventDispatcher\\\\EventDispatcher\",\"type\":\"->\",\"function\":\"dispatch\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/EventDispatcher\\/Debug\\/TraceableEventDispatcher.php\",\"line\":140,\"args\":[[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"]]},{\"namespace\":\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\",\"short_class\":\"TraceableEventDispatcher\",\"class\":\"Symfony\\\\Component\\\\EventDispatcher\\\\Debug\\\\TraceableEventDispatcher\",\"type\":\"->\",\"function\":\"dispatch\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/HttpKernel.php\",\"line\":216,\"args\":[[\"string\",\"kernel.exception\"],[\"object\",\"Symfony\\\\Component\\\\HttpKernel\\\\Event\\\\GetResponseForExceptionEvent\"]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"HttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\HttpKernel\",\"type\":\"->\",\"function\":\"handleException\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/HttpKernel.php\",\"line\":70,\"args\":[[\"object\",\"Symfony\\\\Component\\\\Security\\\\Core\\\\Exception\\\\AccessDeniedException\"],[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"HttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\HttpKernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/DependencyInjection\\/ContainerAwareHttpKernel.php\",\"line\":69,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"],[\"boolean\",true]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\\\\DependencyInjection\",\"short_class\":\"ContainerAwareHttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\DependencyInjection\\\\ContainerAwareHttpKernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/Kernel.php\",\"line\":185,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"],[\"boolean\",true]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"Kernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\Kernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/web\\/app_dev.php\",\"line\":36,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"]]}]},{\"message\":\"Access Denied.\",\"class\":\"Symfony\\\\Component\\\\Security\\\\Core\\\\Exception\\\\AccessDeniedException\",\"trace\":[{\"namespace\":\"\",\"short_class\":\"\",\"class\":\"\",\"type\":\"\",\"function\":\"\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Bundle\\/FrameworkBundle\\/Controller\\/Controller.php\",\"line\":268,\"args\":[]},{\"namespace\":\"Symfony\\\\Bundle\\\\FrameworkBundle\\\\Controller\",\"short_class\":\"Controller\",\"class\":\"Symfony\\\\Bundle\\\\FrameworkBundle\\\\Controller\\\\Controller\",\"type\":\"->\",\"function\":\"createAccessDeniedException\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Bundle\\/FrameworkBundle\\/Controller\\/Controller.php\",\"line\":148,\"args\":[[\"string\",\"Access Denied.\"]]},{\"namespace\":\"Symfony\\\\Bundle\\\\FrameworkBundle\\\\Controller\",\"short_class\":\"Controller\",\"class\":\"Symfony\\\\Bundle\\\\FrameworkBundle\\\\Controller\\\\Controller\",\"type\":\"->\",\"function\":\"denyAccessUnlessGranted\",\"file\":\"\\/srv\\/src\\/Pivot88\\/Bundle\\/RestApiBundle\\/Controller\\/ImportfilesController.php\",\"line\":153,\"args\":[[\"string\",\"ROLE_API\"]]},{\"namespace\":\"Pivot88\\\\Bundle\\\\RestApiBundle\\\\Controller\",\"short_class\":\"ImportfilesController\",\"class\":\"Pivot88\\\\Bundle\\\\RestApiBundle\\\\Controller\\\\ImportfilesController\",\"type\":\"->\",\"function\":\"postImportfilesAction\",\"file\":null,\"line\":null,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"]]},{\"namespace\":\"\",\"short_class\":\"\",\"class\":\"\",\"type\":\"\",\"function\":\"call_user_func_array\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/HttpKernel.php\",\"line\":139,\"args\":[[\"array\",[[\"object\",\"Pivot88\\\\Bundle\\\\RestApiBundle\\\\Controller\\\\ImportfilesController\"],[\"string\",\"postImportfilesAction\"]]],[\"array\",[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"]]]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"HttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\HttpKernel\",\"type\":\"->\",\"function\":\"handleRaw\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/HttpKernel.php\",\"line\":62,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"HttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\HttpKernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/DependencyInjection\\/ContainerAwareHttpKernel.php\",\"line\":69,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"],[\"boolean\",true]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\\\\DependencyInjection\",\"short_class\":\"ContainerAwareHttpKernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\DependencyInjection\\\\ContainerAwareHttpKernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/vendor\\/symfony\\/symfony\\/src\\/Symfony\\/Component\\/HttpKernel\\/Kernel.php\",\"line\":185,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"],[\"string\",\"1\"],[\"boolean\",true]]},{\"namespace\":\"Symfony\\\\Component\\\\HttpKernel\",\"short_class\":\"Kernel\",\"class\":\"Symfony\\\\Component\\\\HttpKernel\\\\Kernel\",\"type\":\"->\",\"function\":\"handle\",\"file\":\"\\/srv\\/web\\/app_dev.php\",\"line\":36,\"args\":[[\"object\",\"Symfony\\\\Component\\\\HttpFoundation\\\\Request\"]]}]}]}}\n"} {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":119,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelResponse"}
[2017-08-23 10:58:41] rest_api.operation.DEBUG: Request to controller Pivot88\Bundle\RestApiBundle\Controller\ImportfilesController::postImportfilesAction has been received.  {"file":"/srv/src/Pivot88/Bundle/RestApiBundle/EventListener/LoggingListener.php","line":90,"class":"Pivot88\\Bundle\\RestApiBundle\\EventListener\\LoggingListener","function":"onKernelController"}
[2017-09-05 04:34:27] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-09-05 04:34:27] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-09-05 04:34:27] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-09-05 04:34:48] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff2\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-09-05 04:34:48] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.woff\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
[2017-09-05 04:34:48] rest_api.operation.ERROR: An exception occurred Symfony\Component\Debug\Exception\FlattenException. {"Code":0,"Message":"No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")","File":"/srv/app/cache/prod/classes.php","Line":2605,"Previous":"[object] (Symfony\\Component\\Debug\\Exception\\FlattenException: {})","Trace":"Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException: No route found for \"GET /bundles/pivot88quality/fonts/fontawesome-webfont.ttf\" (from \"http://localhost:8080/app_dev.php/css/1df1ec4_main_1.css\")\n at .(classes.php:2605)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2605)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)\nCaused by: Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException: \n at .(appProdUrlMatcher.php:2933)\n at appProdUrlMatcher.match(appProdUrlMatcher.php:2933)\n at Symfony.Component.Routing.Matcher.UrlMatcher.matchRequest(classes.php:1435)\n at Symfony.Component.Routing.Router.matchRequest(classes.php:1298)\n at Symfony.Component.HttpKernel.EventListener.RouterListener.onKernelRequest(classes.php:2589)\n at .call_user_func()\n at Symfony.Component.EventDispatcher.EventDispatcher.doDispatch(classes.php:2361)\n at Symfony.Component.EventDispatcher.EventDispatcher.dispatch(classes.php:2276)\n at Symfony.Component.HttpKernel.HttpKernel.handleRaw(bootstrap.php.cache:3199)\n at Symfony.Component.HttpKernel.HttpKernel.handle(bootstrap.php.cache:3172)\n at Symfony.Component.HttpKernel.DependencyInjection.ContainerAwareHttpKernel.handle(bootstrap.php.cache:3323)\n at Symfony.Component.HttpKernel.Kernel.handle(bootstrap.php.cache:2513)\n at (main)(app.php:31)"} []
